reference/safe-core-sdk/onramp-kit/monerium.md

```diff
@@ -0,0 +1,151 @@
+## Monerium
+
+The Monerium pack enables you to take advantage of Safe with Monerium services. [Monerium](https://monerium.com) is a regulated platform that facilitates the use of e-money tokens on the blockchain.
```

Comment:
We should probably add a link to a source that explains e-money tokens or briefly explain e-money tokens in one sentence.

```suggestion
The Monerium pack enables you to use Safe with [Monerium](https://monerium.com), a regulated platform that facilitates the use of e-money tokens on the blockchain.
```